yaml/screenconnect.yaml

```diff
@@ -56,7 +54,23 @@ Artifacts:
   - File: C:\ProgramData\ScreenConnect Client*\user.config
     Description: ScreenConnect client user configuration
     OS: Windows
-  EventLog: []
+  EventLog:
+  - EventID: 20
+    ProviderName: ScreenConnect
+    LogFile: Application.evtx
+    Data: Logs network information (e.g. connection created successfully, connection attempt failed)
+  - EventID: 100
+    ProviderName: ScreenConnect
+    LogFile: Application.evtx
+    Data: User connected
+  - EventID: 101
+    ProviderName: ScreenConnect
+    LogFile: Application.evtx
+    Data: User disconnected
+  - EventID: 200
+    ProviderName: ScreenConnect
+    LogFile: Application.evtx
+    Data: Executed command on host
```

Comment:
Just FYI the fields aren't standard but are related to the provider, so no "Service Name" for example because that's related to EID 7045.
In this case you need to provide the message that is shown in the event log in the data field or a specific field.

For me to evaluate this i need to see the logs. The blog that you linked in the reference doesn't contain refs to those EIDs. So please if you could provide the data for me to review that would be great.